.github/workflows/make-release-commit.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Check out main
       uses: actions/checkout@v4
       with:
-        ref: main
```

Comment:
Should we revert this back?

Reply:
> Should we revert this back?

it's main branch by default, this enables us to run with a specified branch